tiny_dnn/io/tensorflow/tensorflow/core/framework/proto_parser.h

```diff
@@ -0,0 +1,270 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+#include "tensorflow/core/framework/graph.pb.h"
```

Comment:
it's supposed that's generated during the cmake configuration by the proto compiler